simulator/infra/ports/module.h

```diff
@@ -9,9 +9,11 @@
  
 #include <infra/log.h>
 #include <infra/ports/ports.h>
-
 #include <unordered_set>
 
+//Forward declarations
```

Comment:
Please remove the comment.
Comments should highlight something very important that is not obvious from the code itself. Here it is clearly contains declarations